ceno_cli/src/commands/run.rs

```diff
@@ -100,6 +100,9 @@ impl CmdInner {
             .target_selection
             .canonicalize(manifest_path, &self.package_selection)?;
 
+        // XXX: custom handling: set release mode from compilation_options
+        self.ceno_options.release = self.compilation_options.release;
```

Comment:
passing compilation_options where it needs instead add a new field `release` in ceno_options

Reply:
Do you mean add `compilation_options` as an argument to `compilation_options.run()`

https://github.com/scroll-tech/ceno/blob/75d75ea49c5a0c9a9f4383561eaa95cf003b7b19/ceno_cli/src/commands/run.rs#L117-L125

I feel it break the symmetric of other api. For now ceno_options.release mark as `#[clap(skip)]` so I think it's self-explained

Reply:
but this make me feels like "multiple source of the knowledge"